pkg/scheduler/plugins/util/k8s/snapshot.go

```diff
@@ -25,39 +25,39 @@ import (
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/sets"
-	schedulerlisters "k8s.io/kubernetes/pkg/scheduler/listers"
-	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
```

Comment:
the lister and nodeinfo is not available in 1.19?

Reply:
Yes, they are removed